src/Microsoft.Azure.WebJobs.Extensions.Dapr/Utils/LoggingUtils.cs

```diff
@@ -0,0 +1,41 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.WebJobs.Extensions.Dapr.Utils
+{
+    /// <summary>
+    /// Utility methods for logging.
+    /// </summary>
+    public static class LoggingUtils
```

Comment:
Do we really need this class, this seems to be all constants, which we could move to constants.cs class.

Reply:
There are non constants also, right? Like 

```c#
public static string CreateDaprTriggerCategory(string triggerType)
{
    return $"Host.Triggers.Dapr.{triggerType}";
}
```